src/index.html

```diff
@@ -1,13 +1,54 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <div class="card" data-qa="card">
+      <img class="card__img"
+           alt="imac"
+           src="images/imac.jpeg">
+
+      <div class="card__link">
+        <a>APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</a>
+      </div>
+
+      <div class="card__code">
+        Product code: 195434
+      </div>
+
+      <div class="card__grade">
+        <div class="card__grade-stars">
+          <div class="stars-4">
+            <div class="stars-4__star"></div>
+            <div class="stars-4__star"></div>
+            <div class="stars-4__star"></div>
+            <div class="stars-4__star"></div>
+            <div class="stars-4__star"></div>
+          </div>
+        </div>
+        <div class="card__grade-reviews">
+          Reviews: 5
+        </div>
+      </div>
+
+      <div class="card__price">
+        <div class="card__price-text">
+          <span>Price:</span>
+        </div>
+        <div class="card__price-dollars">
+          <span>$2,199</span>
+        </div>
+      </div>
+
+      <div class="card__buy" data-qa = "hover">
+        <a class = "card__buy-buy">BUY</a>
```

Comment:
not needed using <a> here
also class naming `card__buy-buy` looks confusing(you'll remove it with tag <a> but for future try to name more meaningful)